front/pages/w/[wId]/welcome.tsx

```diff
@@ -247,7 +247,19 @@ export default function Welcome({
         headerTitle="Joining Dust"
         headerRightActions={<></>}
       >
-        <Confetti wind={0.02} gravity={0.08} />
+        <Confetti
```

Comment:
Can we use the `ConfettiBackground` here?

Reply:
That's the idea.
But it needs to be deployed first.